src/main/java/gravitas/command/FindCommand.java

```diff
@@ -0,0 +1,37 @@
+package gravitas.command;
+
+import gravitas.exception.DukeException;
+import gravitas.storage.Storage;
+import gravitas.tasklist.TaskList;
+
+/**
+ * Represents a command to find tasks in the task list.
+ */
+public class FindCommand extends Command {
+
+    private static final String EMPTY_FIND = "OOPS!!! The keyword to find cannot be empty.";
+    private String userInput;
+
+    /**
+     * Constructor for FindCommand.
+     *
+     * @param userInput User input to be used by the command.
+     */
+    public FindCommand(String userInput) {
+        this.userInput = userInput;
+    }
+
+    /**
```

Comment:
for javadocs, explain what does string return i.e @return returns the output of all the lists find by the task list